File: src/main/java/com/matag/game/MatagGameApplication.java
package com.matag.game;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.scheduling.annotation.EnableScheduling;

@SpringBootApplication
@EnableScheduling
public class MatagGameApplication {
  public static void main(String[] args) {
    SpringApplication.run(MatagGameApplication.class, args);
  }
}

File: src/main/kotlin/com/matag/game/MatagGameApplication.kt
package com.matag.game

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.scheduling.annotation.EnableScheduling

@SpringBootApplication
@EnableScheduling
open class MatagGameApplication

fun main(args: Array<String>) {
    runApplication<MatagGameApplication>(*args)
}